include/git2/diff.h

```diff
@@ -603,7 +603,7 @@ typedef struct {
 	int    new_lineno;   /**< Line number in new file or -1 for deleted line */
 	int    num_lines;    /**< Number of newline characters in content */
 	size_t content_len;  /**< Number of bytes of data */
-	git_off_t content_offset; /**< Offset in the original file to the content */
+	git_object_size_t content_offset; /**< Offset in the original file to the content */
```

Comment:
Does it really make sense to use `git_object_size_t` here? Diffs aren't necessary referring to any objects at all but, so it seems a bit dirty to me to mix these up.

Reply:
Hmm, perhaps not.  The rare good use of `git_off_t`?

Reply:
Fixed.